barretenberg/cpp/src/barretenberg/benchmark/commit_bench/commit.bench.cpp

```diff
@@ -0,0 +1,38 @@
+
+#include "barretenberg/commitment_schemes/commitment_key.hpp"
+#include <benchmark/benchmark.h>
+
+namespace bb {
+
+template <typename Curve>
+std::shared_ptr<honk::pcs::CommitmentKey<Curve>> create_commitment_key(const size_t num_points)
+{
+    std::string srs_path;
+    if constexpr (std::same_as<Curve, curve::BN254>) {
+        srs_path = "../srs_db/ignition";
+    } else {
+        static_assert(std::same_as<Curve, curve::Grumpkin>);
+        srs_path = "../srs_db/grumpkin";
+    }
+    std::shared_ptr<bb::srs::factories::CrsFactory<Curve>> crs_factory(
+        new bb::srs::factories::FileCrsFactory<Curve>(srs_path, num_points));
+    return std::make_shared<honk::pcs::CommitmentKey<Curve>>(num_points, crs_factory);
+}
+
+constexpr size_t MAX_LOG_NUM_POINTS = 24;
+constexpr size_t MAX_NUM_POINTS = 1 << 24;
```

Comment:
Did you mean to use `MAX_LOG_NUM_POINTS` here instead of 24?

Reply:
Oh yes should have done, thanks